Examples/Python/src/Geant4Component.cpp

```diff
@@ -97,6 +98,88 @@ PYBIND11_MODULE(ActsPythonBindingsGeant4, mod) {
     ACTS_PYTHON_STRUCT_END();
   }
 
+  {
+    using Config = SensitiveSurfaceMapper::Config;
+    auto sm =
+        py::class_<SensitiveSurfaceMapper,
+                   std::shared_ptr<SensitiveSurfaceMapper>>(
+            mod, "SensitiveSurfaceMapper")
+            .def(py::init([](const Config& cfg, Acts::Logging::Level level) {
+              return std::make_shared<SensitiveSurfaceMapper>(
+                  cfg, getDefaultLogger("SensitiveSurfaceMapper", level));
+            }));
+
+    auto c = py::class_<SensitiveSurfaceMapper::Config>(sm, "Config")
```

Comment:
```suggestion
    auto c = py::class_<Config>(sm, "Config")
```

Reply:
We ask in the matching for center() which takes a GeometryContext, the Matcher should get one, actually correctly also the setup of the geometry should use one.